app/models/assignment.rb

```diff
@@ -525,7 +515,7 @@ def summary_json(user)
       if self.assign_graders_to_criteria
         assigned_criteria = user.criterion_ta_associations
                                 .where(assessment_id: self.id)
-                                .pluck(:criterion_type, :criterion_id)
+                                .pluck(:criterion_id)
                                 .map { |t, id| "#{t}-#{id}" }
```

Comment:
This is another line that is no longer necessary: `assigned_criteria` should just be an array of ids. (Modify usage as necessary)